palette/script.js

```diff
@@ -0,0 +1,163 @@
+function CurrentColor() {
```

Comment:
use lowercase for the function names, uppercase for the constructor functions

Reply:
ok. fixed